src/lib/tvgRender.h

```diff
@@ -37,7 +37,7 @@ struct Surface
     uint32_t* buffer;
     uint32_t  stride;
     uint32_t  w, h;
-    uint32_t  cs;
+    uint32_t  colorSpace;
```

Comment:
This is actually out of this goal. Next time please commit an individual refactoring change before the original mission.

Reply:
reverted the code